src/routes/obs.delivery.js

```diff
@@ -1,6 +1,14 @@
-const KoaRouter = require('@koa/router');
+/**
+ * --------------------------------------
+ * --------------------------------------
+ * ---------TO BE DEPRICIATED------------
+ * --------------------------------------
+ * --------------------------------------
+ */
+
 const path = require('path');
-const { DestProxyController } = require('../controllers/destinationProxy');
+const KoaRouter = require('@koa/router');
+const { DestProxyController } = require('../controllers/obs.delivery');
```

Comment:
Just asking, what does `obs` mean in this context ?. Feel free to resolve the comment, once you're done providing explanation

Reply:
It means Obsolete we have https://github.com/rudderlabs/rudder-transformer/blob/8e26b5046b5756e33bc4350f7798715300d5f64a/src/controllers/delivery.ts#L11 as replacement for these